components/core/tools/scripts/lib_install/openssl.sh

```
@@ -0,0 +1,43 @@
+
+echo "🔧 OpenSSL version: $version"
+echo "📁 Install prefix: $prefix"
+echo "📂 OpenSSL directory: $openssldir"
+
+echo "Checking for elevated privileges..."
+if [ ${EUID:-$(id -u)} -ne 0 ] ; then
+  sudo echo "Script can elevate privileges."
+fi
+
+# Download and extract
+cd ${temp_dir}
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)